src/pages/kb/data-sources/querying-urls.md

```diff
@@ -6,30 +6,16 @@ slug: json
 toc: true
 ---
 
-Sometimes you need to visualize data not contained in an RDBMS or NOSQL data store. For those times, Redash provides two URL-based mechanisms to pull in your data: `URL` and `JSON`. Either source can be added from the settings screen in Redash.
+Sometimes you need to visualize data not contained in an RDBMS or NOSQL data store, but available from some HTTP API. For those times, Redash provides the `JSON` data source (first introduced in Redash v8).
 
-{% callout info %}
+Redash treats all incoming data from the `JSON` data source as text; so you should be prepared to use 
```

Comment:
> ... should be prepared to use <eol>

Prepared to use what?  Regex?  Or similar to line 111, planning to mention **Query Results Data Source**?

Reply:
Good catch @yzorg. Looks like Arik chopped this sentence by accident. I restored it.